package.json

```diff
@@ -62,8 +62,9 @@
     "prep": "run-s prep-no-core-rollup rollup",
     "prepublishOnly": "run-s license-badges copy prep build-docs",
     "cypress:open-no-start": "cypress open",
+    "cy:instrument": "rimraf instrumented && cp -r src instrumented && nyc instrument --compact false src instrumented",
```

Comment:
Indeed, my mistake. As the previous "instrument" script didn't work, I used actually a sample from the cypress documentation (by the way, the advantage with the refactoring is that src folder is a common practice and many samples work with no changes...).  -> will follow with a commit to fix this. It will actually remove several scripts not necessary anymore => but please double check this in the upcoming commit.